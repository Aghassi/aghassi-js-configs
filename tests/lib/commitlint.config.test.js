const commitlintConfig = require('../../lib/commitlint.config');
const validateConfig = require('./__helpers__/configSnapshot');

// Use the helper suite to generate a test suite to run
validateConfig('commitlint-config', commitlintConfig);
