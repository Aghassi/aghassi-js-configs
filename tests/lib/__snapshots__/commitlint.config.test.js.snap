// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commitlint-config config matches default config snapshot 1`] = `
Object {
  "extends": Array [
    "@commitlint/config-conventional",
  ],
}
`;
